### PR TITLE
Fix issues with `sqlalchemy.orm.exc.ObjectDeletedError`

### PR DESCRIPTION
**Pull request by doanac**

### CODE REVIEW

**Comment by doanac** [comment]
works on gavelci.net and is deployed in fioed.io for anyone to try.

**Comment by doanac** [comment]
ci-retest

**Comment by doanac** [comment]
ci-retest

File: jobserv/api/worker.py [hunk 1]
```diff
@@ -55,7 +55,6 @@ def wrapper(*args, **kwargs):
                 worker = Worker(w.name, "?", 1, 1, "?", "", 1, w.allowed_tags)
                 worker.enlisted = True
                 db.session.add(worker)
```
Comment (mike-sul) [inline jobserv/api/worker.py:57]: If a worker "registers" itself then why do we need to call `db.session.add(worker)` at all?

File: jobserv/worker.py [hunk 2]
```diff
@@ -240,4 +241,3 @@ def run_monitor_workers():
             time.sleep(120)  # run every 2 minutes
     except Exception:
         log.exception("unexpected error in run_monitor_workers")
-
```
Comment (mike-sul) [inline jobserv/worker.py:243]: Seems like the last empty line is removed.

File: jobserv/api/worker.py (hunk 1 above)
Comment (doanac) [reply jobserv/api/worker.py:57]: scale workers using JWT "register" by calling the update worker API. `db.session.add` is the way you tell the ORM to create a new entry in the DB (once db.session.commit is called)

Comment (doanac) [reply jobserv/api/worker.py:57]: its a bit like the device-gateway where we create some devices on the fly

**Review by mike-sul — APPROVED** [review]

**Pull request merged**